frontend/src/scenes/insights/Insight.scss

```diff
@@ -212,6 +212,10 @@
         padding-right: 1rem;
         min-height: 48px;
     }
+
+    .insight-more-btn-highlight {
```

Comment:
This doesn't really feel like a useful user experience. A yellow background by itself didn't really make me think that there was information behind the button. And when I clicked the button, I saw no tooltip at all – because I'd have to hover, but nobody does that over that over "more options" buttons. You just click it to open the menu.
I propose a slightly different solution: No yellow background. Instead the tooltip is shown at all times, until the user acknowledges it by opening the menu. We should also probably only highlight this as a change for users with `date_joined` before 13 Feb 2023 or so. What do you think?